packages/marko-web/package.json

```diff
@@ -26,6 +26,7 @@
     "@parameter1/base-cms-tenant-context": "^3.0.0",
     "@parameter1/base-cms-utils": "^3.0.0",
     "@parameter1/base-cms-web-common": "^3.3.1",
+    "cheerio": "^0.22.0",
```

Comment:
Use one of the versions that are already installed in this project, `cheerio@^1.0.0-rc.10` or `cheerio@^1.0.0-rc.2`. This should have caused changes to the `yarn.lock` file, which I'm not seeing included here either.